cluster/gce/gci/configure-helper-common.sh

```diff
@@ -3257,6 +3257,7 @@ function wait-until-mizar-ready {
   done
 }
 
+# mizar-operator will be deployed after rp sesrver ready
```

Comment:
The issue is with waiting for RP ready not apply mizar operator. Prefer to have mizar component deployment in a single place.

Also can try call start-mizar-scaleout after all components started.

Reply:
updated all mizar to start  after tp and rp ready